### ISSUE TITLE
What should the structure of an element page be?

### ISSUE THREAD

**Issue by Bjwebb** [IATI/IATI-Extra-Documentation#40]
Currently this is (each section may not appear if it is empty):
- Full element path
- From the schema
- Attributes
- Subelements
- Notes
- Example usage
- Changelog

**Comment by stevieflow** [comment]
Note to talk to @jonihillman and @iatisupport about this

**Comment by stevieflow** [comment]
Proposed:
- Full element path
- From the schema (label: Description)
- Attributes
- Notes
- Example usage
- Who is using this element - link to dashboard?

Marked as in the "footer" of the content pane:
- For developers (link to source - although could be a button)
- Changelog
- Subelements (putting this at the footer of the page as the list can be long)

**Comment by Bjwebb** [comment]
I've put subelements at the bottom. At the moment all the Extra-Documentation is one contiguous block, and it would require a bit of work to split this up. Therefore the page structure is currently the following:
- Full element path
- From the schema
- Attributes
- Notes
- Example usage
- Changelog
- For developers
- Subelements

Where everything from "Notes" to "Changelog" is pulled directly from the Extra-Documentation (so it needs to be in the correct order there)

"Who is using this element - link to dashboard" requires some development work to create - so I've created a separate issue for this https://github.com/IATI/IATI-Standard-SSOT/issues/38 - I don't think this is one we actually need to get done for 1.04, although it would be nice if we did.

Not sure what you're suggestion by "footer of the content pane"

I'm quite happy to rename sections. I think "Definition" might be better than "Description"  for the text taken from the schema. Are you also suggesting we change "Developer tools" to "For developers"? I'm happy to do so.
